terraform/core/main.tf

```diff
@@ -188,6 +189,7 @@ resource "google_app_engine_standard_app_version" "fmltc-app-v1" {
   env_variables = {
     PROJECT_ID = var.project_id
     ORIGIN = "https://${var.project_id}.uc.r.appspot.com"
+    USE_OIDC = "true"
```

Comment:
Do you need to set REDIS_IP_ADDR here? Is REDIS_IP_ADDR only for local use?

Reply:
Yes.  That's in a820c6809cf8893ee579a6f02f7270c9ee73045c